aksel.nav.no/website/pages/eksempler/heading/large.tsx

```diff
@@ -10,7 +10,9 @@ const Example = () => {
       <Divider />
       <div>
         <Descriptor>Spacing </Descriptor>
```

Comment:
Forslag (for å gjøre det tydeligere)
```suggestion
        <Descriptor>Med spacing:</Descriptor>
```
Foreslår også å ikke ha fet skrift. Litt vanskelig å forklare hvorfor jeg synes det ser bedre ut, men teksten blir liksom enklere å skille fra selve overskriften.

Reply:
Godt forslag, oppdaterte eksempler!